satellite/satellitedb/satellitedbtest/run.go

```diff
@@ -26,16 +26,17 @@ const (
 
 // Database describes a test database
 type Database struct {
-	Name    string
-	URL     string
-	Message string
+	Name         string
+	URL          string
+	ForPointerDB bool
```

Comment:
`ForPointerDB` doesn't indicate which database it will use. Let's say we add BigTable support, how would it look like?

Reply:
hmm if we want to more generic we should extend this struct a little bit? e.g. something like:
```
// SatelliteDatabases maybe name can be better
type SatelliteDatabases struct {
	MasterDB  Database
	PointerDB Database
}

// Database describes a test database
type Database struct {
	Name            string
	URL             string
	Message         string
	SkipTestOnEmpty bool
}
```

Reply:
Why `SkiptTestOnEmpty`? But yeah, looks good.

Reply:
Ok, now I looked more into code and looks it won't be necessary.